runtime/onert/core/src/ir/OperationDumper.cc

```diff
@@ -65,6 +81,32 @@ void dumpPackingOp(const Operation &node)
   VERBOSE(LIR) << "  - Inputs : Inputs(" << inputs << ")" << std::endl;
   VERBOSE(LIR) << "  - Output : Output(" << node.getOutputs().at(0) << ")" << std::endl;
 }
+
+std::string genInputSeqStr(const Operation &node)
+{
+  std::string inputs;
+  const auto &input_indices = node.getInputs();
+  for (auto it = std::begin(input_indices); it != std::end(input_indices); ++it)
+  {
+    inputs += std::to_string(it->value());
+    if (std::next(it) != std::end(input_indices))
+      inputs += ", ";
+  }
+  return inputs;
+}
+
+std::string genOutputSeqStr(const Operation &node)
+{
+  std::string outputs;
+  const auto &output_indices = node.getOutputs();
+  for (auto it = std::begin(output_indices); it != std::end(output_indices); ++it)
+  {
+    outputs += std::to_string(it->value());
+    if (std::next(it) != std::end(output_indices))
+      outputs += ", ";
+  }
+  return outputs;
+}
```

Comment:
What about using 1 function instead of  `genInputSeqStr` and `genOutputSeqStr`? Or we may use `std::ostream &operator<<(std::ostream &o, const OperandIndexSequence &operand_seq) `.